ern-container-gen-android/src/AndroidGenerator.ts

```diff
@@ -707,10 +701,51 @@ You should replace "${annotationProcessorPrefix}:${dependency}" with "annotation
           'lib/src/main/java/com/walmartlabs/ern/container/plugins',
         );
         shell.cp(pathToPluginConfigHook, pathToCopyPluginConfigHookTo);
+
+        if (semver.gte(rnVersion, '0.60.0')) {
+          const filesConverted = this.convertToAndroidX(
+            pathToCopyPluginConfigHookTo,
+          );
+          if (filesConverted > 0) {
+            log.info(
+              `${plugin.name} contains source files with references to the Android Support Library (android.support.*)`,
+            );
+            log.info(
+              `${filesConverted} files successfully converted to use AndroidX (androidx.*)`,
+            );
+          }
+        }
       }
     }
   }
 
+  /**
+   * Convert files in a directory from support library to AndroidX
+   * eg: import android.support.annotation.NonNull => import androidx.annotation.NonNull
+   */
+  public convertToAndroidX(dir: string): number {
+    const filesWithSupportLib: string[] = [];
+    shell.pushd(dir);
+    shell
+      .ls('-R', '.')
+      .filter((file) => file.match(/\.(java|kt)$/))
+      .forEach((file) => {
+        if (shell.grep('android.support', file).trim().length !== 0) {
+          filesWithSupportLib.push(file);
+        }
+      });
+
+    if (filesWithSupportLib.length !== 0) {
```

Comment:
I don't think this `if` check here is necessary. `forEach` on an empty array will do the expected thing (nothing), or am I wrong?